### PR TITLE
Added write_gcm to README file.

### PR DESCRIPTION
**Pull request by jkohen**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -462,6 +462,10 @@ Features
       needed. Please read collectd-unixsock(5) for a description on how that's
       done.
 
+    - write_gcm
+      Sends data to Stackdriver, Google's Monitoring solution for applications
```
Comment (igorpeshansky) [inline README]: Sends data to Stackdriver Monitoring, ...

Comment (jkohen) [reply README]: I prefer the current wording. "Sends data to Stackdriver Monitoring, Google's Monitoring solution for applications" sounds redundant to me. Do you have a different suggestion?

Thanks for pointing out the other location. We should update it to mention Stackdriver. What do you think?

Comment (igorpeshansky) [reply README]: I'm just a bit concerned that this implies that Stackdriver is a monitoring (-only) solution. But I may be imagining things. We could use the text on the [Stackdriver pages](https://cloud.google.com/stackdriver/) for inspiration...

Yes, it's a good idea to update the other location as well, though let's be careful to leave the explanation of why it's named `write_gcm`...

Comment (jkohen) [reply README]: PTAL.

**Pull request merged**